mephisto/abstractions/blueprints/static_html_task/source/dev/app.jsx

```diff
@@ -126,10 +142,16 @@ function ShowURL({ url, data = null }) {
     requestTaskHMTL(url).then((data) => setRetrievedHtml(data));
   }, []);
 
-  return <HtmlRenderer html={retrievedHtml} data={data} />;
+  return (
+    <HtmlRenderer
+      html={retrievedHtml}
+      data={data}
+      mephisto_keys={mephisto_keys}
+    />
+  );
 }
 
-function HtmlRenderer({ html, data }) {
+function HtmlRenderer({ html, data, mephisto_keys }) {
```

Comment:
overall looks good, I'd just recommend using the JS camelCase, e.g. `mephistoKeys`.

also, thought on naming it as  `mephistoVars` instead? i feel that keys is a little data-structure/implementation specific, whereas vars denotes that they are values set via mephisto